db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150416231106) do
+ActiveRecord::Schema.define(version: 20150511231229) do
```

Comment:
I had a few questions about this schema.rb, wether it should be commited or not. After double checking that with Shane, I had to clean it up because of the way how Boxen bootstraps the projects. All Zendesk projects were using the same database, which screwed up this file. All tables, from all projects, were added to this file. :-P

I'll commit the file ASAP.